<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lb_serverIP">
          <property name="text">
           <string>Server IP </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_serverIP">
          <property name="placeholderText">
           <string>IP adress</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_serverPort">
          <property name="text">
           <string>Server Port </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_serverPort">
          <property name="placeholderText">
           <string>Port</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_identifiant">
          <property name="text">
           <string>Identifiant</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_identifiant">
          <property name="placeholderText">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_connectDisconnect">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lb_clients">
          <property name="text">
           <string>                                                                 Connected clients</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_clients">
          <property name="minimumSize">
           <size>
            <width>191</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QListWidget" name="lw_recvChat"/>
        </item>
        <item>
         <widget class="QPushButton" name="pb_ClearRecvChat">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="lw_trmtChat"/>
        </item>
        <item>
         <widget class="QPushButton" name="pb_ClearTrmtChat">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_msgChat">
          <property name="placeholderText">
           <string>Your message</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>616</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
